libraries/abi/Filesystem.h

```diff
@@ -27,18 +27,45 @@ enum FileType
     FILE_TYPE_TERMINAL,
 };
 
+struct Mode
+{
+public:
+    bool u_r;
```

Comment:
You should do a union and add a `as_uint` form (that will make this bit:  (see bellow) easier to read)

```c++
Mode _mode = {true, true, true, true, true, true, true, true, true};
```

Reply:
Done